### PR TITLE
Add BehatCommands for Drush 9

### PR DESCRIPTION
**Pull request by greg-1-anderson**
Requires https://github.com/drush-ops/drush/pull/3687, which I will merge shortly.

### CODE REVIEW

**Comment by greg-1-anderson** [comment]
~Refactored everything. Maybe I should have just stuck with #11, but I sort of have thoughts of making some improvements while I'm in here.~

Went back to @arosboro's implementation

File: BehatCommands.php [hunk 1]
```diff
@@ -0,0 +1,202 @@
+<?php
+namespace Drush\Commands\example_site_wide_command;
```
Comment (jhedstrom) [inline BehatCommands.php]: This should probably be updated to `BehatDrushEndpoint` or similar?

File: BehatCommands.php [hunk 2]
```diff
@@ -0,0 +1,202 @@
...
+
+    /**
+     * Expands properties on the given entity object to the expected structure.
+     *
+     * @param \stdClass $entity
+     *   Entity object.
+     *
+     * @see Drupal\Driver\Cores\AbstractCore\expandEntityFields
+     */
+    function expandEntityFields($entity_type, \stdClass $entity) {
```
Comment (jhedstrom) [inline BehatCommands.php]: This and the other methods that don't already should probably specify visibility (`protected`, `public`, etc) for clarity.

**Review by jhedstrom — COMMENTED** [review]
This looks good. Just these nits.

Want to open a PR against the DrupalExtension that targets this branch of the behat drush endpoint to see if tests start passing again? There's this related issue https://github.com/jhedstrom/drupalextension/issues/458

File: BehatCommands.php (hunk 1 above)
Comment (greg-1-anderson) [reply BehatCommands.php]: It will be `behat_drush_endpoint`

**Comment by greg-1-anderson** [comment]
See https://github.com/jhedstrom/drupalextension/pull/506

**Pull request merged**